db/migrate/20150804194147_create_blacklisted_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBlacklistedEvents < ActiveRecord::Migration
+  def change
+    create_table :blacklisted_events do |t|
+      t.string  :event_name
+      t.string  :provider_model
+      t.bigint  :ems_id
+      t.boolean :system
+      t.boolean :enabled
+      t.timestamps
```

Comment:
This has added a bunch of noise to the migration specs...

> DEPRECATION WARNING: `#timestamps` was called without specifying an option for `null`. In Rails 5, this behavior will change to `null: false`. You should manually specify `null: true` to prevent the behavior of your existing migrations from changing. (called from block in change at `db/migrate/20150804194147_create_blacklisted_events.rb:9`)